app/models/app_suggestion_ticket.rb

```diff
@@ -7,6 +7,11 @@ class AppSuggestionTicket
                 :name,
                 :email
 
+  validates :details, presence: { message: "Fill in your suggestion" }
+  validates :reply, presence: { message: "Select a reply option" }
+  validates :email, email: { message: "The email address must be valid" }, if: :can_reply?, allow_blank: true
```

Comment:
Another not a blocker, but: might be worth doing a quick check of what error messages we use in other places for this error case, as email validity is a bit of a tricksy subject.

Reply:
Ah, I see there was a content review in commit 14 that dealt with this.